crypto/fipsmodule/service_indicator/service_indicator_test.cc

```diff
@@ -1619,6 +1620,41 @@ TEST_P(HKDF_ServiceIndicatorTest, HKDFTest) {
   EXPECT_EQ(approved, test.expect_approved);
 }
 
+// TODO(CryptoAlg-1281): Do we need to test HKDF_Expand separately from this?
+class EVP_HKDF_ServiceIndicatorTest : public TestWithNoErrors<HKDFTestVector> {};
```

Comment:
I suggest we set the mode to `EXTRACT_AND_EXPAND` as in https://github.com/awslabs/aws-lc/blob/b0017d27ccc47339ab0a51f144e071545b12b026/crypto/fipsmodule/hkdf/hkdf_test.cc#L348 and test that's Approved.
Then set it to `EXTRACT_ONLY` and `EXPAND_ONLY` and make sure that's Not_Approved.

Reply:
done!